decals-cutout.py

```diff
@@ -35,13 +35,29 @@
                         help='output verbosity')
     parser.add_argument('-z','--zoom',default=12,type=int,
                         help='zoom level')
+    parser.add_argument('-t','--table',help='table for easyaccess')
     args = parser.parse_args()
 
     if args.database == 'dessci':
         # Query the DESDM database
         import easyaccess as ea
```

Comment:
I think it is probably better now to put this in a try... except at the top of the file:
```
try:
    import easyaccess as ea
except ImportError as e:
    print("Warning: easyaccess not found")
    pass
```
or you can "print(e.msg)"